### PR TITLE
Update CLI

### PR DESCRIPTION
**Pull request by junha1**
#385 depends on this

### CODE REVIEW

File: cli/src/cli.rs [hunk 1]
```diff
@@ -137,12 +136,27 @@ pub enum Commands {
...
     /// Show the current status of the p2p network.
     Network,
+    /// Show the status of the Simperby repository.
+    ///
+    /// It checkes the following for the current directory:
+    ///
+    /// 1. Is it a valid Git repository?
+    /// 2. Does it contain a valid `.simperby/` directory?
+    /// 3. Does it have a valid `reserved/` directory?
+    /// 4. Does it have all the reserved branches and tags?
```
Comment (TomTaehoonKim) [inline cli/src/cli.rs:150]: I think it might be tricky for overlapping use of "reserved".

File: cli/src/cli.rs [hunk 2]
```diff
@@ -154,6 +168,14 @@ pub enum Commands {
...
     Update,
     /// Broadcast relevant data to the p2p network.
+    ///
+    /// This may contain
+    ///
+    /// 1. Newly created commits
+    /// 2. Consensus progress
+    /// 3. Governance votes
+    /// 5. Known peers
+    /// 6. 1-5 of the above, sent from other peers.
```
Comment (TomTaehoonKim) [inline cli/src/cli.rs:178]: How about fixing it to "All of the above"?

**Review by TomTaehoonKim — CHANGES_REQUESTED** [review]

File: cli/src/cli.rs [hunk 3]
```diff
@@ -137,12 +136,27 @@ pub enum Commands {
...
+    /// Show the status of the Simperby repository.
+    ///
+    /// It checkes the following for the current directory:
+    ///
+    /// 1. Is it a valid Git repository?
+    /// 2. Does it contain a valid `.simperby/` directory?
+    /// 3. Does it have a valid `reserved/` directory?
+    /// 4. Does it have all the reserved branches and tags?
+    /// 5. Does the reserve state at `finalized` branch match the block header?
+    /// 6. Does the reserve state at `work` branch match the block header?
```
[the remark this answers is not in the document]
Comment (junha1) [reply cli/src/cli.rs:152]: Fixed. Btw 6 was wrong. I updated.

File: cli/src/cli.rs (hunk 1 above)
Comment (junha1) [reply cli/src/cli.rs:150]: Updated to `protected`

File: cli/src/cli.rs (hunk 2 above)
Comment (junha1) [reply cli/src/cli.rs:178]: fixed

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by just-do-halee — APPROVED** [review]

**Pull request merged**